src/System.Management.Automation/engine/Modules/GetModuleCommand.cs

```diff
@@ -573,9 +573,11 @@ private void GetLoadedModules(string[] names, IDictionary<string, ModuleSpecific
             IDictionary<string, ModuleSpecification> moduleSpecTable,
             PSModuleInfo module)
         {
+            const WildcardOptions options = WildcardOptions.IgnoreCase | WildcardOptions.CultureInvariant;
```

Comment:
Got this code from https://github.com/PowerShell/PowerShell/blob/5d98b901be24094f96cad6b00bfcade96735e468/src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs#L1077
which is called earlier in the code path.